assets/sass/_pulumi.scss

```diff
@@ -323,10 +315,6 @@ pre {
     margin-bottom: 24px;
 }
 
-pre.highlight {
-    border-radius: 2px !important;
```

Comment:
This is not necessary as the block above sets the `border-radius` for `pre` to `2px`.